lucille-core/src/main/java/com/kmwllc/lucille/connector/storageclient/GoogleStorageClient.java

```diff
@@ -162,17 +172,15 @@ private Document blobToDoc(Blob blob, InputStream content, String decompressedFu
     }
 
     // unable to get decompressed file size
-    if (getFileContent) {
+    if (params.shouldGetFileContent()) {
       doc.setField(FileConnector.CONTENT, content.readAllBytes());
     }
 
     return doc;
   }
 
-  public static void validateOptions(Map<String, Object> cloudOptions) {
-    if (!cloudOptions.containsKey(GOOGLE_SERVICE_KEY)) {
-      throw new IllegalArgumentException("Missing " + GOOGLE_SERVICE_KEY + " in cloudOptions for GoogleStorageClient.");
-    }
+  public static boolean validOptions(Config cloudOptions) {
```

Comment:
same comment as elsewhere:
this method seems to only be called from within validateOptions()
please move this logic inside validateOptions and eliminate this static validOptions method